vms/platformvm/state/camino_test.go

```diff
@@ -1,22 +1,23 @@
-// Copyright (C) 2022, Chain4Travel AG. All rights reserved.
+// Copyright (C) 2022-2023, Chain4Travel AG. All rights reserved.
```

Comment:
Ouch!? Why? Do we have some legal opinion to bump years in CC?
I'm against such modifications unless it's necessary because it simply adds meaningless work.

Reply:
It is necessary, because it contains work from 2023, no?

Reply:
To answer when & why file was changed is the git/github job, IMO. This is just a suggestion, I'm fine when ignored.
 Hint: some big tech companies ignores the year part of the CC 👐